doc/configuration.rst

```diff
@@ -399,6 +406,24 @@ All available configuration options for Ansible are:
 
       * ``env``: An optional list environment variables to set when creating the container. The format of each variable is ``var_name=value``.
 
+.. _option_ansible_config_cfg:
+
+  * ``config``: This is a complex structure allowing to define `Ansible configuration settings <https://docs.ansible.com/ansible/latest/reference_appendices/config.html>`_  if you need a specific Ansible Configuration.
+    
+    * You should first provide the Ansible Configuration section (for example ``defaults``, ``inventory``, ``ssh_connection``...).
+    * You should then provide the list of parameters within this section, ie. what `Ansible documentation <https://docs.ansible.com/ansible/latest/reference_appendices/config.html>`_ describes as the ``Ini key`` within the ``Ini Section``.
+      Each parameter value must be provided here as a string : for a boolean parameter, you would provide the string ``False`` or ``True`` as expected in Ansible Confiugration.
+      For example, it would give in Yaml:
+
+.. code-block:: YAML
+
+  ansible:
+    config:
+      defaults:
+        display_skipped_hosts: "False"
+        special_context_filesystems: "nfs,vboxsf,fuse,ramfs,myspecialfs"
+        timeout: "60"
+
```

Comment:
Default parameters should appear in the documentation and a visible warning should say that modifying them may lead to unpredictable results...

Reply:
ok! added a section describing ansible config settings defined by Yorc, and a warning directive about the potential impact on overringd these settings